### DISCUSSION TITLE
How to change order of questions/slides?

### DISCUSSION THREAD

**Discussion by anzepintar** [mawoka-myblock/ClassQuiz#305] [Q&A] [answered]
Is it possible to change the order of questions/slides?

**Comment by mawoka-myblock** [comment]
That's not possible (yet)

**Reply by anzepintar** [reply]
I can export the quiz as .cqa, which is just a compressed json, and change the order of slides. But how could I compress it back into the .cqa?

**Reply by mawoka-myblock** [reply, marked as answer]
.cqa is a bit more complex, since it also contains the images. If your quiz only contains voting and multiple choice questions, you can export it as a spreadsheet, edit it and import it again

**Reply by anzepintar** [reply]
Thanks, this works.
Not everything, but I can export excel, change questions and then import the file back. It automatically updates the quiz (except colors and images).
